judo-runtime-core-jsl-itest/src/test/java/hu/blackbelt/judo/runtime/core/jsl/QueryTest.java

```diff
@@ -42,124 +35,57 @@
-        assertOrderByDescending(MyEntityWithOptionalFields.Attribute.BOOL_ATTR, entity1);
-        assertOrderByDescending(MyEntityWithOptionalFields.Attribute.DATE_ATTR, entity1);
-        assertOrderByDescending(MyEntityWithOptionalFields.Attribute.TIMESTAMP_ATTR, entity1);
-        assertOrderByDescending(MyEntityWithOptionalFields.Attribute.TIME_ATTR, entity1);
-        assertOrderByDescending(MyEntityWithOptionalFields.Attribute.ENUM_ATTR, entity1);
-    }
+    @Inject
+    Lead.LeadDao leadDao;
 
-    @Test
-    public void testMask() {
-        MyEntityWithOptionalFields maskedResult = myEntityWithOptionalFieldsDao
-                .query()
-                .maskedBy(MyEntityWithOptionalFields.MyEntityWithOptionalFieldsDao.Mask.myEntityWithOptionalFieldsMask()
-                        .withStringAttr()
-                        .withIntegerAttr())
-                .filterByStringAttr(StringFilter.equalTo("test"))
-                .execute()
-                .get(0);
-
-        assertEquals(entity1.get__identifier(), maskedResult.get__identifier());
-        assertEquals(entity1.getIntegerAttr(), maskedResult.getIntegerAttr());
-        assertEquals(Optional.empty(), maskedResult.getScaledAttr());
-        assertEquals(entity1.getStringAttr(), maskedResult.getStringAttr());
-        assertEquals(Optional.empty(), maskedResult.getRegexAttr());
+    @Override
+    public Module getModelDaoModule() {
+        return new QueryModelDaoModules();
     }
 
-    private void assertOrderBy(MyEntityWithOptionalFields.Attribute attribute, MyEntityWithOptionalFields firstEntity) {
-        MyEntityWithOptionalFields orderBy = myEntityWithOptionalFieldsDao
-                .query()
-                .orderBy(attribute)
-                .execute()
-                .get(0);
-
-        assertEquals(firstEntity.get__identifier(), orderBy.get__identifier());
+    @Override
+    public String getModelName() {
+        return "QueryModel";
     }
 
-    private void assertOrderByDescending(MyEntityWithOptionalFields.Attribute attribute, MyEntityWithOptionalFields firstEntity) {
-        MyEntityWithOptionalFields orderByDescending = myEntityWithOptionalFieldsDao
-                .query()
-                .orderByDescending(attribute)
-                .execute()
-                .get(0);
-
-        assertEquals(firstEntity.get__identifier(), orderByDescending.get__identifier());
+    @Test
+    public void testStaticQuery() {
+        leadDao.create(Lead.builder().withValue(50).build());
+        leadDao.create(Lead.builder().withValue(175).build());
+
+        assertEquals(2, totalNumberOfLeadsDao.getTotalNumberOfLeads());
+        assertEquals(2, rootAllLeadsDao.getRootAllLeads().size());
+        assertNotNull(rootOneLeadDao.getRootOneLead());
+
+        List<Lead> rootAllLeadsBetween = rootAllLeadsBetweenDao.searchRootAllLeadsBetween()
+                .execute(_QueryModel_rootAllLeadsBetween_Parameters.builder()
```

Comment:
I suggest that we create a ticket to generate a bit more user friendly name

Reply:
https://blackbelt.atlassian.net/browse/JNG-4206